src/main/java/net/imagej/ops/topology/EulerCharacteristic26N.java

```diff
@@ -0,0 +1,112 @@
+package net.imagej.ops.topology;
+
+import net.imagej.ops.Contingent;
+import net.imagej.ops.Op;
+import net.imagej.ops.special.function.AbstractUnaryFunctionOp;
+import net.imglib2.RandomAccess;
+import net.imglib2.RandomAccessibleInterval;
+import net.imglib2.type.BooleanType;
+import net.imglib2.view.Views;
+import org.scijava.plugin.Plugin;
+
+/**
+ * An Op which calculates the euler characteristic (χ) of the given 3D binary image.
+ * Here Euler characteristic is defined as χ = β_0 - β_1 + β_2, where β_i are so called Betti numbers.
+ * β_0 = number of separate particles
+ * β_1 = number of handles
+ * β_2 = number enclosed cavities
+ * <p>
+ * The Op calculates χ by using the triangulation algorithm described by Toriwaki & Yonekura (see below).
+ * There it's calculated X = ∑Δχ(V), where V is a 2x2x2 neighborhood around each point in the 3D space.
+ * We are using the 26-neighborhood version of the algorithm. The Δχ(V) values here are predetermined.
+ * <p>
+ * For the algorithm see
+ * Toriwaki J, Yonekura T (2002) Euler Number and Connectivity Indexes of a Three Dimensional Digital Picture.
+ * Forma 17: 183-209.
+ * <a href="http://www.scipress.org/journals/forma/abstract/1703/17030183.html">
+ * http://www.scipress.org/journals/forma/abstract/1703/17030183.html</a>
+ * <p>
+ * For the Betti number definition of Euler characteristic see
+ * Odgaard A, Gundersen HJG (1993) Quantification of connectivity in cancellous bone,
+ * with special emphasis on 3-D reconstructions.
+ * Bone 14: 173-182.
+ * <a href="http://dx.doi.org/10.1016/8756-3282(93)90245-6">doi:10.1016/8756-3282(93)90245-6</a>
+ *
+ * @author Richard Domander
+ * @author David Legland (original MatLab implementation)
+ */
+@Plugin(type = Op.class, name = "eulerCharacteristic26N")
```

Comment:
The goal is for every commit to compile with passing tests, as described in the [Reproducible builds](http://imagej.net/Architecture#Reproducible_builds) section of the web site.

So, best would be to squash commits that fix the namespace into this initial commit.